src/main/java/org/broadinstitute/dsde/consent/ontology/datause/services/TextTranslationServiceImpl.java

```diff
@@ -89,6 +89,13 @@ public String translateDataset(String dataUseString) throws IOException {
         return translate(dataUse, TranslateFor.DATASET);
     }
 
+    @Override
+    public String translateParagraph(String dataUseString) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
`ObjectMapper` is thread safe so most applications only need to instantiate one copy. In an app that uses IOC (inversion of control) (that is, one that uses `Inject` or `Autowired`) the usual approach is to make it a singleton that can be injected, in case you want to configure it specially for you app.

Reply:
As a side note, we've been trying to migrate away from using Jackson libraries, in general. I much prefer Gson for this kind of job.

Reply:
When you have time I'd be interested to know why you prefer gson over Jackson. The main reason I use Jackson is because most DSP java apps use Spring, and Spring uses Jackson, and I'd rather use the same library as spring (since my app is already importing that library) unless I have a good reason not to. And so far I haven't run into any issues or shortcomings using Jackson.

Reply:
I have two issues with Jackson. One is that it had been a large source of security vulnerabilities and required many frequent updates. That has subsided over the past year so is less of an issue today. The other one is the amount of boilerplate code necessary with model class annotations. I find Gson far more flexible from that perspective.